### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mtrilbybassett**
msal-net is not a supported product taxonomy

## Purpose
Making modifications to the configurations of Readme.md files to be accepted by samples processor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,6 @@ page_type: sample
 languages:
 - csharp
 products:
-- msal-net
```
Comment (Dickson-Mwendia) [inline README.md:6]: Let's use msalnet instead, I see its on the [taxonomies allowlist](https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main#product) under dotnet

**Review by Dickson-Mwendia — CHANGES_REQUESTED** [review]
@mtrilbybassett minor update here before we merge this.

File: README.md (hunk 1 above)
Comment (mtrilbybassett) [reply README.md:6]: changed to msalnet

**Review by Dickson-Mwendia — APPROVED** [review]
LGTM!

**Pull request merged**